cobbler/items/profile.py

```diff
@@ -471,7 +471,7 @@ def filename(self, filename: str):
                 filename = ""
         self._filename = filename
 
-    @property
+    @InheritableProperty
```

Comment:
Maybe I'm missing something but shouldn't we also make this function to return `self._resolve('autoinstall')` here https://github.com/cobbler/cobbler/pull/3358/files#diff-ed38504ace636301eed8cc1a20fe77a1f15ec6b413e37bac50904b56bb76046bR494 ?

Reply:
Correct. This needs to be `_resolve()`. Will push a tmp commit and see what the tests are saying.